kolibri/plugins/utils/__init__.py

```diff
@@ -481,7 +481,7 @@ def check_plugin_config_file_location(version):
 def iterate_plugins():
     # Use to dedupe plugins
     plugin_ids = set()
-    for entry_point in entry_points().get("kolibri.plugins", []):
+    for entry_point in entry_points().select(group="kolibri.plugins"):
```

Comment:
Sorry, the task was insufficiently clear here - this needs to be done in a way that detects whether the output of entry points has a `select` method - in that case select should be used, otherwise the existing `get` should be used (this is why you are seeing test failures on earlier versions of Python).

Reply:
Correct as usual. I apologise my thoughts are very constrained for now when it comes to contributing to big projects please bear with me 😅